### ISSUE TITLE
I'm getting: Invalid prop: type check failed for prop "tabindex" on all link clicks

### ISSUE THREAD

**Issue by liambarr** [noxify/gridsome-starter-articles#2]
Upon installing and trying to run gridsome develop - I get:
```
These dependencies were not found:
* core-js
```

Running `npm i core-js` lets me run gridsome develop however has lead to a new error.
Now any time I click a link, page and everything appear to be interacting correctly - but terminal shows:

``` 
vue.runtime.esm.js?2b0e:619 [Vue warn]: Invalid prop: type check failed for prop "tabindex". Expected Number with value 0, got String with value "0".

found in

---> <BackToTop>
       <DefaultLayout> at src/layouts/DefaultLayout.vue
         <Index> at src/pages/Index.vue
           <App> at node_modules/gridsome/app/fallbacks/App.vue
             <Root>
```

And it's forever ongoing as I interact with the site.

**Comment by noxify** [comment]
Hi @liambarr 

thanks for reporting.

I haven't had the time to test it with the latest gridsome version.

If you were able to solve it, feel free to send a PR with a fix :)

**Comment by noxify** [comment]
will be fixed with #4
